core/src/test/java/bio/terra/pearl/core/service/kit/KitRequestServiceTest.java

```diff
@@ -118,14 +125,14 @@ void testUpdateKitStatus(TestInfo testInfo) throws Exception {
         String sentDate = "2023-11-17T14:57:59.548Z";
         String errorMessage = "Something went wrong";
         PepperKit pepperKit = PepperKit.builder()
-            .juniperKitId(kitRequest.getId().toString())
-            .currentStatus(PepperKitStatus.SENT.pepperString)
-            .scanDate(sentDate)
-            .errorMessage(errorMessage)
-            .build();
+                .juniperKitId(kitRequest.getId().toString())
+                .currentStatus(PepperKitStatus.SENT.pepperString)
+                .scanDate(sentDate)
+                .errorMessage(errorMessage)
+                .build();
```

Comment:
Everything looks good, I'm wondering if the whitespace changes are correct here? (I'm not implying that they are not, just asking)

Reply:
Honestly I have no idea what is preferred, I think this was just my default IDE settings. I think I changed it to where my IDE now should only reformat lines that changed , so I should have less of these random extra formatting changes in the future.